src/NavBar/StyleMenu/StyleMenu.js

```diff
@@ -15,49 +16,77 @@ class StyleMenu extends Component {
         }
     }
 
-    renderStylingMenu() {
-        if (this.state.shouldDisplayStyleMenu) {
-            return (
-                <div>
-                    <div className="style_thumb" onClick={()=>this.props.changeDisplayStyle('Thumb')}>
-                        <FontAwesomeIcon icon="list-ul"  className="icon_list"/>Thumb
-                    </div>
-                </div>
-            )
-        }
-        return null;
+    renderListButton() {
+        return (
+            <div className="style_thumb" 
+                onClick={()=>{this.props.changeDisplayStyle('List'); 
```

Comment:
Executing multiple functions in an unanonymous function is unintuitive. Please group into one function